### ISSUE TITLE
Multi-tenancy support 

### ISSUE THREAD

**Issue by dahabit** [cartalyst/sentinel#435]
Hello guys, it's has been a long time now we are waiting for Sentinel 3 and Multi-tenancy support any news about that or is there are links explain how we can use other packages like https://github.com/hyn/multi-tenant work correctly with Sentinel

**Comment by dahabit** [comment]
@bkintanar thanks for your replay ,  my question is when you enabled multi-tenant do you had to extend Sentinel core classes to add DB connection traits or what? I'm using  https://github.com/AsgardCms/Platform

**Comment by bkintanar** [comment]
@dahabit apparently yes. 😞

**Comment by breaktag** [comment]
I am also trying to implement Sentinel too. All my user credentials are stored on each tenant

I have tried using the `force-tenant-connection-of-models` config option with no luck sadly:

```
/**
 * Define a list of classes that you wish to force onto the tenant or system connection.
 * The connection will be forced when the Model has booted.
 *
 * @info Useful for overriding the connection of third party packages.
 */
'force-tenant-connection-of-models' => [
    \Cartalyst\Sentinel\Activations\EloquentActivation::class,
    \Cartalyst\Sentinel\Persistences\EloquentPersistence::class,
    \Cartalyst\Sentinel\Reminders\EloquentReminder::class,
    \Cartalyst\Sentinel\Roles\EloquentRole::class,
    \Cartalyst\Sentinel\Throttling\EloquentThrottle::class,
    \Cartalyst\Sentinel\Users\EloquentUser::class
]
```

Does anyone have any tips or suggesstions?

**Comment by bkintanar** [comment]
@breaktag are you still using hyn/multi-tenant? there's a new package called tenancy/tenancy which is more robust and powerful.

**Comment by breaktag** [comment]
@bkintanar Yes, although I thought it would still be possible. I will give the new package ago anyway
